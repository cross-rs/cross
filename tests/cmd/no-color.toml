bin.name = "cross"
args = ["--color"]
status = "failed"
