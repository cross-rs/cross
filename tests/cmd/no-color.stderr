[..][cross] error[..]: The argument '[..]--color <WHEN>[..]' requires a value but none was supplied
               [possible values: [..]auto[..], [..]always[..], [..]never[..]]

USAGE:
    cross [+toolchain] [OPTIONS] [SUBCOMMAND]

For more information try [..]--help[..]
